modules/oneVideoBidAdapter.js

```diff
@@ -25,14 +25,12 @@ export const spec = {
       return false;
     }
 
-    // Prevend DAP Outstream validation
+    // Prevend DAP Outstream validation, Banner DAP validation & Multi-Format adUnit support
     if (bid.mediaTypes.video) {
       if (bid.mediaTypes.video.context === 'outstream' && bid.params.video.display === 1) {
         return false;
       }
-    }
-    // Banner DAP validation
-    if (bid.mediaTypes.banner && (!bid.params.video.display)) {
+    } else if (bid.mediaTypes.banner && !bid.params.video.display) {
```

Comment:
Please check bid.params.video before going into bid.params.video.display

Reply:
Hi @bretg bid.params.video are checked previously on line 24.
The Validation would fail there before even reaching the Banner DAP validations if they were missing